### PR TITLE
completed ticket 7

### PR DESCRIPTION
**Pull request by manheicatherine**
Hello,
I completed ticket 7! Please check. Thank you!

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -145,4 +144,66 @@ describe("API Testing", () => {
...
+              })
+            );
+          });
+        });
+    });
+
+    test("Error handling 1: passing a invalid username but existing id", () => {
+      return request(app)
+        .post("/api/articles/5/comments")
+        .send({ username: "Blackpink", body: "Blackpick in your area~" })
```
Comment (HeatherEver) [inline __tests__/app.test.js:184]: 10/10 for a good comment 👏

File: models/articles.models.js [hunk 2]
```diff
@@ -32,12 +32,28 @@ exports.fetchArticlesById = (id) => {
...
-        }else if (rows.length === 0) {
+exports.addComment = (id, username, body) => {
+  return db
+    .query(
+      "INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;",
+      [id, username, body]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+      } else if (rows.length === 0) {
```
Comment (HeatherEver) [inline models/articles.models.js]: this top if statement is empty and checking the same thing as your else if statement - we don't need both, we can just have the if statement checking the rows.length here

**Review by HeatherEver — APPROVED** [review]
Nice work Catherine - just one comment to have a look at 😊

File: models/articles.models.js (hunk 2 above)
Comment (manheicatherine) [reply models/articles.models.js]: oh, haha mistake!! Thanks for your feedback!

**Pull request merged**